packages/react-testing/src/root.tsx

```diff
@@ -201,16 +205,19 @@ export class Root<Props> implements Node<Props> {
   trigger<K extends FunctionKeys<Props>>(
     prop: K,
     ...args: DeepPartialArguments<Props[K]>
-  ): ReturnType<
-    NonNullable<
-      Props[K] extends ((...args: any[]) => any) | undefined ? Props[K] : never
-    >
-  > {
+  ): NonNullable<Props[K]> extends (...args: any[]) => any
+    ? ReturnType<NonNullable<Props[K]>>
+    : any {
     return this.withRoot((root) => root.trigger(prop, ...(args as any)));
   }
 
-  triggerKeypath<T = unknown>(keypath: string, ...args: unknown[]) {
-    return this.withRoot((root) => root.triggerKeypath<T>(keypath, ...args));
+  triggerKeypath<
```

Comment:
Same as previous comment - Having a `keypath` argument here ensured that users must specify at least one argument. By removing the keypath argument, you've made it so `element.triggerKeypath();` is considered valid code from a type checking perspective.

`element.triggerKeypath();` should raise a type error as the first value - the keypath is required. Please retain that behaviour.